.travis.yml

```diff
@@ -26,6 +26,14 @@ matrix:
     - stage: Test
       script: make helm-sync-test
       env: HELM_SYNC_TEST=true
+    - stage: Deploy
+      if: type = push AND env(DOCKER_USERNAME) IS present
+      script: make sync-readme-to-dockerhub
```

Comment:
should we be doing this on every push? can we detect if there was actually a change to the readme before replacing?